src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1302,14 +1305,20 @@ def mark_privacy_request_pre_approve_eligible(
         webhook.key,
         webhook.connection_config.key,
     )
-    PreApprovalWebhookReply.create(
-        db=db,
-        data={
-            "webhook_id": webhook.id,
-            "privacy_request_id": privacy_request_id,
-            "is_eligible": True,
-        },
-    )
+    try:
+        PreApprovalWebhookReply.create(
+            db=db,
+            data={
+                "webhook_id": webhook.id,
+                "privacy_request_id": privacy_request_id,
+                "is_eligible": True,
+            },
+        )
+    except Exception as exc:
+        raise HTTPException(
+            status_code=HTTP_500_INTERNAL_SERVER_ERROR,
```

Comment:
Can we give this a status other than 500?

Reply:
yeah lemme think what makes most sense. Some kind of 400 or 422?